sqlmesh/core/engine_adapter/base.py

```diff
@@ -1466,23 +1466,29 @@ def remove_managed_columns(
         if check_columns:
             row_check_conditions = []
             for col in check_columns:
-                t_col = col.copy()
+                col_qualified = col.copy()
+                col_qualified.set("table", exp.parse_identifier("joined"))
```

Comment:
just curious: shouldn't this just be `exp.to_identifier` since there isn't much parsing to speak of